server/app/services/Utils.java

```diff
@@ -0,0 +1,27 @@
+package services;
+
+import play.Configuration;
+
+import java.io.*;
+
+public class Utils {
+
+  public static JiraInfo getJiraInfo(Configuration configuration){
+   return new JiraInfo(configuration.getString("jira.account"), configuration.getString("jira.pwd"));
+  }
+
+  public static void writeMissedQuery(String missedQuestion){
+    try {
+      BufferedWriter out = new BufferedWriter
+        (new FileWriter("logs/missedQuery.log", true));
```

Comment:
If a method is unused then remove the method and all references to it from the codebase.